### PR TITLE
Mention the today button when the applyValueMode is useButtons

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

**Review by sometimesElen — CHANGES_REQUESTED** [review]
We need to add info that the Today button will be displayed if pickerType: "calendar"

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/applyValueMode.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 The option accepts the following values.
 
 - "useButtons"  
- Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively.
+ Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively. Choosing this mode activates the [showTodayButton](/api-reference/10%20UI%20Widgets/dxCalendar/1%20Configuration/showTodayButton.md '/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#showTodayButton') option, and the *'today'* button appears in the left bottom corner.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDateBox/1 Configuration/applyValueMode.md]: Write button names without quotes and in normal font:

> the Today button

**Review by DevExpressSupportTeam — APPROVED** [review]
LGTM

**Review by sometimesElen — APPROVED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/dxDateBox/1 Configuration/applyValueMode.md [hunk 2]
```diff
@@ -5,10 +5,10 @@
 The option accepts the following values.
 
 - "useButtons"  
- Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively.
+ Enables an end-user to apply the selected value or cancel the selection using the Apply and Cancel buttons respectively. When the [pickerType](/api-reference/10%20UI%20Widgets/dxDateBox/1%20Configuration/pickerType.md '/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#pickerType') is *'calendar'*, choosing this mode activates the [showTodayButton](/api-reference/10%20UI%20Widgets/dxCalendar/1%20Configuration/showTodayButton.md '/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#showTodayButton') option, and the Today button appears.
```
Comment (arminal) [inline api-reference/10 UI Widgets/dxDateBox/1 Configuration/applyValueMode.md]: From your writing, it is not clear how the Today button appearing is related to the rest of the sentence. Please rewrite.

**Pull request merged**